sources/background_scripts/notifications.js

```diff
@@ -0,0 +1,25 @@
+NewTabTweaker = {};
+NewTabTweaker.Notifications = {
+	notification_ids: [],
+	notify: function(notification_id, message){
+		var title = browser.runtime.getManifest().name;
```

Comment:
Since this does not depend on function parameters, and its value is not expected to change, it should be declared outside the function as a `const` variable.